action.yaml

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
Should we run this task?

Reply:
This is not a task, it's the definition of our GHA